### PR TITLE
Finish RSS feeds, share menus, routes refactoring

### PR DESCRIPTION
**Pull request by mattleff**
Finishes https://trello.com/c/vNwxd8aK/80-error-loading-states, https://trello.com/c/oL0zwW3u/85-make-share-buttons-functional. <100 `TODO`s now!

localhost_3000_en_presenters_13_norman-mcnulty html

image

### CODE REVIEW

File: src/containers/presenter/recordings.spec.tsx [hunk 1]
```diff
@@ -68,18 +68,6 @@ describe('presenter recordings page', () => {
 		expect(paths).toContain('/en/presenters/the_presenter_id/page/1');
 	});
 
-	// TODO: add?
```
Comment (narthur) [inline src/containers/presenter/recordings.spec.tsx:71]: Should we have a todo somewhere for adding an RSS link to the list page?

File: src/containers/sermon/list.all.spec.tsx [hunk 2]
```diff
@@ -272,15 +272,15 @@ describe('sermons list page', () => {
 		expect(link.href).toContain('/es/teachings/all/page/1');
 	});
 
-	it('sets rss head link', async () => {
-		loadSermonListData();
+	// it('sets rss head link', async () => {
```
Comment (narthur) [inline src/containers/sermon/list.all.spec.tsx:275]: Shouldn't we continue to have RSS metadata in the document head?

**Review by narthur — APPROVED** [review]
Great work!

File: src/components/HOCs/withFailStates.tsx [hunk 3]
```diff
@@ -14,7 +16,7 @@ const withFailStates = <P extends any>(
 		}
 
 		if (isFallback) {
-			return <h1>Loading…</h1>;
+			return <LoadingCards />;
```
[the remark this answers is not in the document]
Comment (mattleff) [reply src/components/HOCs/withFailStates.tsx:19]: Ideally yes, but I don't think it's necessary for MVP. This should only be visible for first load until the static page is built.

File: package.json [hunk 4]
```diff
@@ -28,10 +28,11 @@
     "clsx": "^1.1.1",
     "cookie": "^0.4.1",
     "dayjs": "^1.10.6",
-    "feed": "^4.2.1",
+    "feed": "github:mattleff/feed#custome-elements-build",
```
[the remark this answers is not in the document]
Comment (mattleff) [reply package.json:31]: Sure: 85c9fe3

File: src/containers/sermon/list.all.spec.tsx (hunk 2 above)
Comment (mattleff) [reply src/containers/sermon/list.all.spec.tsx:275]: Good point: 85c9fe3

File: src/containers/presenter/recordings.spec.tsx (hunk 1 above)
Comment (mattleff) [reply src/containers/presenter/recordings.spec.tsx:71]: The link is available on the presenter share menu, but I've added it to the head with 85c9fe3.

**Pull request merged**